go/enclave/enclave.go

```diff
@@ -723,6 +723,15 @@ func (e *enclaveImpl) DoEstimateGas(args *gethapi.TransactionArgs, blockNr gethr
 	return hexutil.Uint64(hi), nil
 }
 
+// HealthCheck returns whether the enclave is deemed healthy
+func (e *enclaveImpl) HealthCheck() (bool, error) {
+	if e.storage.FetchHeadState() == nil {
```

Comment:
I think it would be cleaner to add a new method. You don't want this to break if the API of this method ever breaks.

Reply:
You mean add a method in the storage ? Yeah that makes sense. What should the method do ? Fetch the latest rollup ?

Reply:
I'd say try and read something. Even if the read returns an empty result (e.g. reading a dummy hash), you know it's successfully able to speak to the DB.